.e2e.env.sample

```diff
@@ -0,0 +1,14 @@
+NEXT_PUBLIC_INFURA_KEY=
+NETWORK_NAME=localhost
+
+# We don't use PRIVATE_KEY because Synpress is looking for that to set up MetaMask
+# Instead we generate the private key in runtime before tests start
+# Below key is only used to fund the newly created wallet
+PRIVATE_KEY_CUSTOM=b6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659
+
+# We set up MetaMask ourselves
+SKIP_METAMASK_SETUP=true
+
+# Private key of the user wallet with enough deposits and withdrawals on Goerli to pass the tx history tests
+# Also funded by E2E_PRIVATE_KEY wallet and used for all the tests
+PRIVATE_KEY_USER=c791f745783e3545f83e7b4f0adcc0c2f591c5c8bc796e042c639824ec5d7bf0
```

Comment:
Do we need any deposits and withdrawals populated in the wallet to run E2E in retryables?

The only reason we use this key on the bridge is because it has some transaction data needed to test tx history.

- If we need some transactions in the wallet for retryables, we can leave it as it is.
- If we don't, it may be better not to pass this over, and instead create a random wallet in `synpress.config.ts` and pass its private key to `config.env.PRIVATE_KEY`. This way to don't unnecessarily overload our test wallet.